packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -96,10 +96,15 @@ BuildApp() {
 
   RunCommand rm -rf -- "${derived_dir}/App.framework"
 
+  local flutter_engine_flag=""
   local local_engine_flag=""
   local flutter_framework="${framework_path}/Flutter.framework"
   local flutter_podspec="${framework_path}/Flutter.podspec"
 
+  if [[ -n "$FLUTTER_ENGINE" ]]; then
```

Comment:
As specified in https://github.com/flutter/flutter/wiki/The-flutter-tool , 
```
Using a locally-built engine with the flutter tool
You can also set the environment variable $FLUTTER_ENGINE instead of specifying --local-engine-src-path.
```
I want to use the unified concept.